Rubberduck.Resources/RubberduckUI.de.resx

```diff
@@ -1509,7 +1509,7 @@ Import abgebrochen.</value>
 Import abgebrochen.</value>
   </data>
   <data name="ImportCommand_DocumentDoesNotExist" xml:space="preserve">
-    <value>Es existiert kein Dokumentenmodul namens '{0}', s.d. die Datei '{1}' nicht importiert werden kann.
+    <value>Es existiert kein Dokumentenmodul namens '{0}', die Datei '{1}' kann nicht importiert werden kann.
```

Comment:
Wenn man den Nebensatz in einen getrennten Hauptsatz umwandeln will, muss man auch das Satzzeichen ändern, das erste Wort groß schreiben und das Verb am Ende löschen. Ich persönlich favorisierte übrigens den Nebensatz, was ja eigentlich auch typisch deutsch ist.